### ISSUE TITLE
Convenience flag in CLI for latest workflow

### ISSUE THREAD

**Issue by rbreeze** [argoproj/argo#3128]
# Summary

When using the CLI there's no easy way to access the last submitted workflow.

# Motivation

When running through the tutorials I found myself typing `argo list` and copy pasting the generated name to `argo get` and `argo logs` a lot. 

# Proposal

Add the ability to append `--latest` to a command to automatically use the last submitted workflow. 

---
**Message from the maintainers**:

If you wish to see this enhancement implemented please add a 👍 reaction to this issue! We often sort issues this way to know what to prioritize.

**Comment by alexec** [comment]
what about `--sort-by=.metadata.creationTimestamp` or similar?

**Comment by alexec** [comment]
btw `kubectl get wf --sort-by=.metadata.creationTimestapm`

**Comment by alexec** [comment]
```
argo get $(kubectl get wf --sort-by=.metadata.creationTimestamp -o name |tail -n1)
```

**Comment by rbreeze** [comment]
I got an error but seem to have fixed it with an awk at the end: 

`argo get $(kubectl get wf --sort-by=.metadata.creationTimestamp | tail -n1 | awk '{print $1}')`

Would it be a good idea to integrate this as a simple "--latest" flag for less typing, or is there not much of a use case for it?

**Comment by hadim** [comment]
Maybe I did something wrong but I can't use the `--latest` flag. I downloaded 2.9.0-rc2 and checked `argo version` returns the correct version. But none of these commands work:

- `argo --latest get`
- `argo get --latest`
- `argo --latest logs`
- `argo logs --latest`

**Comment by simster7** [comment]
It's `@latest`:

* `argo get @latest`
* `argo logs @latest`

etc.

**Comment by simster7** [comment]
Closed in #3179
